.circleci/config.yml

```diff
@@ -949,6 +1014,18 @@ workflows:
               - postgres-integration-test
               - cache-server-test
             filters: *filter-not-release-or-master
+        - grafana-docker-ubuntu-pr:
```

Comment:
Is it necessary to build the ubuntu image for the PRs? Given the overhead, I think it would be better to avoid it if it's not obligatory.

Reply:
@papagian I think @marefr wanted this? At least it gets built in parallel.